api-reference/10 UI Components/dxDataGrid/1 Configuration/onRowClick.md

```diff
@@ -32,7 +32,7 @@ The row's [group index](/api-reference/_hidden/dxDataGridColumn/groupIndex.md '/
 Indicates whether internal UI component functions have already handled the event.
 
 ##### field(e.isExpanded): Boolean
-Indicates whether or not the group row is expanded. Available if **rowType** is *"group"*.
+Indicates whether the row is expanded or collapsed. Available if **rowType** is *"data"*, *"detail"*, or *"group"*.
```

Comment:
The isExpanded field does not existl for 'detail' row because detail row is shown only if row is expanded and this field is not necessary.